apps/client/components/APM.tsx

```diff
@@ -1,19 +1,19 @@
-import { useAuth0 } from '@auth0/auth0-react'
 import { useEffect } from 'react'
 import * as Sentry from '@sentry/react'
+import { useSession } from 'next-auth/react'
 
 export default function APM() {
```

Comment:
What does APM stand for?

Reply:
I believe its Application Performance Monitoring